modules/pam_unix/pam_unix_acct.c

```diff
@@ -183,9 +183,9 @@ int _unix_run_verify_binary(pam_handle_t *pamh, unsigned int ctrl,
  */
 
 int
-pam_sm_acct_mgmt(pam_handle_t *pamh, int flags, int argc, const char **argv)
+pam_sm_acct_mgmt(pam_handle_t *pamh, unsigned long long flags, int argc, const char **argv)
```

Comment:
This is wrong. You cannot change the pam module API function signature.

Reply:
Fixed.